File: WanaD.AppServiceContract/OrderData.cs


using System;
using System.Collections.Generic;

namespace WanaD.AppServiceContract
{
    public class OrderData
    {
        public Guid Id { get; set; }
        public Guid FactoryId { get; set; }
        public Guid ProductTypeId { get; set; }
        public Guid ProductMasterId { get; set; }
        public DateTime? DeliveredDate { get; set; }
        public DateTime? FinishedDate { get; set; }
        public decimal Price { get; set; }
        public decimal UnitPrice { get; set; }
        public string OrderStatus { get; set; }
        public string Remark { get; set; }
        public string OrderNo { get; set; }
        public int Quantity { get; set; }
        public DateTime DueDate { get; set; }
        public DateTime RequestDate { get; set; }
        public string ProductName { get; set; }
        public string ProductTypeName { get; set; }
        public string FactoryName { get; set; }
        public decimal Received { get; set; }
        public decimal Percent { get; set; }
        public ProductMasterData ProductMasterData { get; set; }
        public FactoryData FactoryData { get; set; }
        public string Unit { get; set; }
        public string ProductMasterNo { get; set; }
        public string StatusCode { get; set; }
        public string StatusName { get; set; }
        public Guid? WorkflowDocumentId { get; set; }
        public decimal? PaidAmount { get; set; }
        public bool IsReSubmit { get; set; }
        public string ProductTypeCode {  get; set; }

    }
}

File: WanaD.WebClient/app/src/Modules/WanaD/Scripts/AppServiceContract/OrderData.ts

 

 

 

    import ProductMasterData from 'Modules/WanaD/Scripts/AppServiceContract/ProductMasterData';

    import FactoryData from 'Modules/WanaD/Scripts/AppServiceContract/FactoryData';

 
class OrderData 
{
    
        Id: string;
    
        FactoryId: string;
    
        ProductTypeId: string;
    
        ProductMasterId: string;
    
        DeliveredDate: Date;
    
        FinishedDate: Date;
    
        Price: number;
    
        UnitPrice: number;
    
        OrderStatus: string;
    
        Remark: string;
    
        OrderNo: string;
    
        Quantity: number;
    
        DueDate: Date;
    
        RequestDate: Date;
    
        ProductName: string;
    
        ProductTypeName: string;
    
        FactoryName: string;
    
        Received: number;
    
        Percent: number;
    
        ProductMasterData: ProductMasterData;
    
        FactoryData: FactoryData;
    
        Unit: string;
    
        ProductMasterNo: string;
    
        StatusCode: string;
    
        StatusName: string;
    
        WorkflowDocumentId: string;
    
        PaidAmount: number;
    
        IsReSubmit: boolean;
    
        ProductTypeCode: string;
    
}
 
export default OrderData;
 




